examples/layers/preprocessing/random_color_degeneration_demo.py

```diff
@@ -0,0 +1,49 @@
+"""random_shear_demo.py shows how to use the RandomShear preprocessing layer.
```

Comment:
do we need a license header for this?

Reply:
Updated and updated script to include this.